### ISSUE TITLE
INI: Value 'Raw' of argument 'scanner_mode' is not supported

### ISSUE THREAD

**Issue by Indigo744** [peachpiecompiler/peachpie#826]
Hello,

We have a lot of [`parse_ini_file()` call with `INI_SCANNER_RAW`](https://www.php.net/manual/en/function.parse-ini-string.php), which shows the following warning: 
```
PHP (Warning): Value 'Raw' of argument 'scanner_mode' is not supported
```

The culprit is found here: https://github.com/peachpiecompiler/peachpie/blob/master/src/Peachpie.Library/FileSystem.Ini.cs#L51

What would be the correct approach to handle this flag?

**Comment by jakubmisek** [comment]
I would add the `scanner_mode` parameter to `ArrayBuilder`, and handle it in there. Eventually, add some flags to  `IParserCallbacks` so the parser will behave according to that.

**Comment by Indigo744** [comment]
Alright, I'll look into it.

I identified a bunch of implementation mismatch while adding the test, so I went ahead a create a PR for fixing all of this.

**Comment by jakubmisek** [comment]
@Indigo744 nice! thank you
